Dockerfile

```diff
@@ -36,12 +12,9 @@ RUN pip install -e .[nlp] --no-use-pep517
 
 # ---- test -----
 # we need to install test dependencies before, so we cannot use 'base_baal' as base image
-FROM continuumio/miniconda3 as test_baal
+FROM pytorch/pytorch:1.8.0-cuda11.1-cudnn8-devel as test_baal
```

Comment:
We have `pytorch/pytorch:1.8.1-cuda11.1-cudnn8-devel` available as well, are there any incompatibility issues? Otherwise it should be considered

Reply:
Maybe there is a typo in your message, but we use pytorch/pytorch:1.8.1-cuda11.1-cudnn8-devel